otherlibs/stdune/ansi_color.ml

```diff
@@ -146,6 +146,8 @@ let stdout_supports_color = lazy (supports_color Unix.stdout)
 
 let stderr_supports_color = lazy (supports_color Unix.stderr)
 
+let output_is_a_tty fd = lazy (Unix.isatty fd)
```

Comment:
isn't the point of laziness defeated here? we're creating the thunk and immediately forcing it. it should be `let output_is_tty = lazy (Unix.isatty Unix.stderr)` or we should get rid of the laziness altogether

Reply:
Of course you're right, I mixed the two.